clinlp/entity.py

```diff
@@ -213,3 +214,41 @@ def __call__(self, doc: Doc):
         doc.set_ents(entities=ents)
 
         return doc
+
+
+def create_term(row, col_term):
+    """Creates a Term object with available attributes."""
+    term = Term("")
+    for attr, value in row.items():
+        if attr == col_term:
+            term.phrase = value
+        if value == value:
```

Comment:
Dit is altijd true?

Reply:
Ik heb dit gebruikt om te checken of er een NaN value in het Dataframe zit, volgens: https://stackoverflow.com/questions/50468494/numpy-pandas-clean-way-to-check-if-a-specific-value-is-nan

Martijn vond het er ook al raar uitzien, dus andere suggesties zijn ook welkom.